### DISCUSSION TITLE
Keine Verbindung zum TI RU Endpoint

### DISCUSSION THREAD

**Discussion by MirkoWe** [Digitaler-Impfnachweis/certification-apis#75] [Q&A] [answered]
Der Authentication Flow funktioniert bei uns momentan nur mit dem Endpoint der Testumgebung (https://keycloak-cert-auth.dev.ibmega.net/auth/realms/master/protocol/openid-connect/auth)
Mit dem TI RU Endpoint (https://id.ru.impfnachweis.info/auth/realms/ubirch-certify/protocol/openid-connect/auth) bekomme ich beim holen der challenge folgenden Fehler: 
OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to id.ru.impfnachweis.info:443
Benötigen wir hier noch ein spezielles  X.509 Client Certificate? Wenn ja, woran kann ich mich wenden, um dieses zu beantragen?

**Comment by MirkoWe** [comment, marked as answer]
Hat sich erledigt

**Comment by ManuelB** [comment]
Ich hab mal unsere Lösung hier detailliert erklärt: https://youtu.be/oLrjF-OISO0 https://github.com/ere-health/dgc-ps-app
